src/probnum/problems/_problems.py

```diff
@@ -1 +1,204 @@
-"""Definitions of problems solved by probabilistic numerical methods."""
+"""Definitions of problems currently solved by probabilistic numerical methods.
+
+If a problem is not listed here, we have not implemented a probabilistic
```

Comment:
Petty opinion: This comment seems obsolete to me or at least I would change it to passive voice, i.e. ..., there is no implemented probabilistic numerical method for it yet.